sdcm/cluster.py

```diff
@@ -2040,6 +2040,7 @@ def scylla_setup(self, disks):
         """
         result = self.remoter.run('/sbin/ip -o link show |grep ether |awk -F": " \'{print $2}\'', verbose=True)
         devname = result.stdout.strip()
+        extra = self.parent_cluster.params.get('append_scylla_setup_args')
```

Comment:
`extra` is a bit vague.

maybe `extra_setup_args` would be better ?

Reply:
changes